### ISSUE TITLE
Build emergency/recovery PROD dispatch workflow

### ISSUE THREAD

**Issue by DerekRoberts** [bcgov/devops-quickstart-helpers#12]

**Comment by conbrad** [comment]
Is this a workflow for deploying a previous deploying for recovery?

**Comment by DerekRoberts** [comment]
@conbrad That's right!  Ideally it'd deploy a specific release based on user input.  If not, then whatever images are labeled PROD.

**Comment by conbrad** [comment]
Cool I want this

**Comment by RMCampos** [comment]
That would be super helpful.

**Comment by DerekRoberts** [comment]
@conbrad @RMCampos Moved to sprint backlog.  Thanks for the input.  :)

**Comment by DerekRoberts** [comment]
@conbrad Ideally, yes.  I'd like to build this after tying in releases to deployments.  Container tags aren't quite as good.  No idea about the pre/post hooks, but it's a great idea to brainstorm!
